src/prebuild/prebuild.csproj

```diff
@@ -0,0 +1,44 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="..\..\build\all_projects.settings.targets" />
+  <PropertyGroup>
+    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
+    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
+    <ProjectGuid>{95C0421F-D4C1-46E8-9177-6DAA4D71D0F2}</ProjectGuid>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <GenerateAssembly Include="$(ILDir)*.il">
+      <Visible>false</Visible>
+    </GenerateAssembly>
+  </ItemGroup>
+
+  <Target Name="GenerateAssemblies"
+        Condition="'@(GenerateAssembly)' != ''"
+        Inputs="@(GenerateAssembly)"
+        Outputs="@(GenerateAssembly->'$(GeneratedAssembliesDir)%(FileName).dll')"
+        AfterTargets="Build">
+    <PropertyGroup>
+      <IlAsmCommand>"$(windir)\Microsoft.NET\Framework\v4.0.30319\ilasm.exe"</IlAsmCommand>
```

Comment:
On my box, there is no other copy of ilasm.exe under Program Files, and since it ships with the desktop .NET Framework, I can't think of any reason it wouldn't be there.